crates/biome_plugin_loader/src/analyzer_js_plugin.rs

```diff
@@ -72,7 +73,18 @@ impl AnalyzerJsPlugin {
 }
 
 impl AnalyzerPlugin for AnalyzerJsPlugin {
-    fn evaluate(&self, _root: AnyParse, path: Arc<Utf8PathBuf>) -> Vec<RuleDiagnostic> {
+    fn language(&self) -> PluginTargetLanguage {
+        PluginTargetLanguage::JavaScript
+    }
+
+    fn query(&self) -> Vec<RawSyntaxKind> {
+        AnyJsRoot::KIND_SET
```

Comment:
Maybe a TODO is in order here? I think you want to make JS plugins more granular in scope, right?